numpy/ma/core.pyi

```diff
@@ -230,8 +230,8 @@ _ScalarT_co = TypeVar("_ScalarT_co", bound=generic)
 # A subset of `MaskedArray` that can be parametrized w.r.t. `np.generic`
 _MaskedArray: TypeAlias = MaskedArray[_Shape, dtype[_ScalarT]]
 
-MaskType = bool
-nomask: bool
+MaskType: TypeAlias = bool_
```

Comment:
a `TypeAlias` is not meant to be instantiated at runtime, so it should just be

```suggestion
MaskType = bool_
```

so that it's inferred as `MaskType: type[bool_]`